src/services/todoServices.js

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue';
+
```

Comment:
we can import Axios directly and call it instead of importing whole Vue instance